docs/cudf/source/_ext/PandasCompat.py

```diff
@@ -84,71 +94,114 @@ def merge_PandasCompats(app, env, docnames, other):
         )
 
 
-def process_PandasCompat_nodes(app, doctree, fromdocname):
-    if not app.config.include_pandas_compat:
-        for node in doctree.traverse(PandasCompat):
-            node.parent.remove(node)
+class PandasCompatdDomain(Domain):
```

Comment:
The extra `d` might be a typo?
```suggestion
class PandasCompatDomain(Domain):
```

Reply:
Thanks for the spot